sensor_msgs/Image image
geometry_msgs/Pose cam_pose